specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/virtualNetworkGateway.json

```diff
@@ -362,6 +362,49 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/resetvpnclientsharedkey": {
+      "post": {
+        "tags": [
+          "VirtualNetworkGateways"
+        ],
+        "operationId": "VirtualNetworkGateways_Resetvpnclientsharedkey",
```

Comment:
Could we have a better name? For example `VirtualNetworkGateways_ResetVpnClientSharedKey`

Reply:
Done as requested

Reply:
@sergey-shandar Done as requested